qiskit/algorithms/quantum_time_evolution/imaginary/qite.py

```diff
@@ -0,0 +1,53 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Interface for Quantum Imaginary Time Evolution."""
+
+from abc import abstractmethod
+
+from qiskit.algorithms.quantum_time_evolution.evolution_base import EvolutionBase
+from qiskit.opflow import OperatorBase, StateFn, Gradient
+
+
+class Qite(EvolutionBase):
+    """Interface for Quantum Imaginary Time Evolution."""
+
+    @abstractmethod
```

Comment:
Exactly as you explained, used solely for typing.
We include these methods because otherwise IDE complains that the class does not implement all abstract methods from the parent class (`EvolutionBase`).